### ISSUE TITLE
AWS Lightsail-Wordpress MultiSite Failure

### ISSUE THREAD

**Issue by gregtout** [bitnami/vms#1123]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

ee4ad633-45bc-fb6c-8cdc-bcebd7914d91

### bndiagnostic output

===== Begin of bndiagnostic tool output =====

    ? Apache: Found possible issues
    ? Mariadb: Found possible issues
    ? Connectivity: Found possible issues
    ✓ Wordpress: No issues found
    ? Resources: Found possible issues
    ✓ Processes: No issues found
    ✓ Php: No issues found

[Apache]

Found recent error or warning messages in the Apache error log.
```
[Mon Aug 28 04:32:45.401270 2023] [proxy_fcgi:error] [pid 5811:tid
140465630279424] [client **ip_address**:58880] AH01071: Got error 'PHP message:
Jetpack WAF Blocked Request     block   -1      403     ip block list'
 [Mon Aug 28 04:32:46.584061 2023] [proxy_fcgi:error] [pid 5811:tid
140465495996160] [client **ip_address**:41776] AH01071: Got error 'PHP message:
Press [Enter] to continue:
Jetpack WAF Blocked Request     block   -1      403     ip block list'
 [Mon Aug 28 04:33:47.020677 2023] [authz_core:error] [pid 5811:tid
140465705813760] [client **ip_address**:6230] AH01630: client denied by server
configuration: /opt/bitnami/apache/cgi-bin, referer:
http://simplybots.com/cgi-bin/moon.php

```
Please check the following guide to troubleshoot server issues:

https://docs.bitnami.com/general/apps/wordpress/troubleshooting/debug-erro
rs-apache/

[Mariadb]

Found recent error messages in the MariaDB error log:
```
2023-08-27 16:06:44 17178 [Warning] Aborted connection 17178 to db:
'bitnami_wordpress' user: 'bn_wordpress' host: 'localhost' (Got an error reading
communication packets)

```
Please check the following guide to troubleshoot MariaDB issues:

Press [Enter] to continue:
https://docs.bitnami.com/aws/apps/wordpress/troubleshooting/debug-errors-m
ariadb/

[Connectivity]

Server ports 22, 80 and/or 443 are not publicly accessible. Please check the
following guide to open server ports for remote access:

https://docs.bitnami.com/general/faq/administration/use-firewall/

[Resources]

Your instance has little available RAM memory.
```
 total used free shared buff/cache available Mem: 987 607 64 84 314 136 Swap:
634 87 547
```
You could try to increase your instance's memory. Please check your cloud
provider's documentation for more information.

===== End of bndiagnostic tool output =====

### bndiagnostic was not useful. Could you please tell us why?

I checked the apache error log file and I am not seeing any memory errors.

### Describe your issue as much as you can

The server every couple (2 to 3 days) stops responding to http/https requests.  When I run ctlscript.sh status when the server is locked up I receive running statuses:

bitnami@ip-172-26-11-130:/opt/bitnami/apache/logs$ sudo /opt/bitnami/ctlscript.sh status
apache already running
mariadb already running
php-fpm already running

What I did notice is the listener on ports 80 and 443 appears to be non-responsive based on Recv-Q:

Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      -
tcp        0      0 127.0.0.1:3306          0.0.0.0:*               LISTEN      -
tcp6       0      0 :::22                   :::*                    LISTEN      -
tcp6     129      0 :::443                  :::*                    LISTEN      -
tcp6      45      0 :::80                   :::*                    LISTEN      -
udp        0      0 127.0.0.1:323           0.0.0.0:*                           -
udp        0      0 0.0.0.0:68              0.0.0.0:*                           -
udp6       0      0 ::1:323                 :::*                                -
udp6       0      0 fe80::b0:91ff:feef::546 :::*     

To clear the issue I run sudo /opt/bitnami/ctlscript.sh restart and the server is stable for a couple of days before it locks up again.  Any suggestions for addressing this issue will be greatly appreciated.

Thank you for your help.

**Comment by mdhont** [comment]
According to the bndiagnostic tool it also seems the machine is running low on resources. I recommend upgrading it to a more powerful one.

**Comment by gregtout** [comment]
Does the tool provide you with what resources (RAM, CPU, etc) the machine is low on? If so, please recommend a configuration for the server.  Thank you.

**Comment by mdhont** [comment]
Yes, it's low on RAM. Which in case of WordPress is usually because of one or more plugin. I recommend disabling all of them and then enable them one by one to see if it makes any difference in memory usage.

**Comment by gregtout** [comment]
Just a follow-up to this issue.

I moved the site to a new VM with 2 gigs of RAM and 2 vCPUs but the issue still continued to occur.  I then disabled and deleted Jetpack from the network and the site has not dropped since.  So there must have been an issue with JetPacks interaction with one of the other plugins I am using which are:

Divi
All in One SEO
LiteSpeed Cache
Real Simple SSL

Hope the above info helps someone else from banging their head against a wall.
